rest-api/config/config_stable.json

```diff
@@ -52,7 +52,8 @@
       "roles": ["awardee_sa"],
       "awardee": "ILLINOIS",
       "whitelisted_ip_ranges" : {
-        "ip4": ["10.50.178.54", "165.124.250.40", "165.124.250.41", "131.193.169.188", "165.124.199.1/28"]
+        "ip4": ["10.50.178.54", "165.124.250.40", "165.124.250.41", "131.193.169.188", "165.124.199.1/28"],
+        "ip6": ["2604:2000:f660:3c00:b9a0:919a:d2be:329c"]
```

Comment:
Is there a ticket for this that we can double check this address against?

Reply:
https://precisionmedicineinitiative.atlassian.net/browse/ROC-46

https://precisionmedicineinitiative.atlassian.net/browse/PD-2950

The address is in the images they uploaded in the ROC ticket